cirq/aqt/aqt_sampler.py

```diff
@@ -0,0 +1,226 @@
+        op_str = 'MS'
+    elif isinstance(op_obj, XPowGate):
+        op_str = 'X'
+    elif isinstance(op_obj, YPowGate):
+        op_str = 'Y'
+    else:
+        raise RuntimeError('Got unknown gate:', op_obj)
+    return op_str
+
+
+class AQTSampler(Sampler):
+    """Sampler for the AQT ion trap device
+    This sampler connects to the AQT machine and
+    runs a single circuit or an entire sweep remotely
+    """
+
+    def _run_api(
+            self,
+            circuit: circuits.Circuit,
+            param_resolver: Sweepable,
+    ) -> str:
+        """
+        Args:
+            circuit: Circuit to be run
+            param_resolver: Param resolver for the
+
+        Returns:
+            json formatted string of the sequence
+        """
+
+        seq_list = []
+        circuit = resolve_parameters(circuit, param_resolver)  # type: ignore
+        # TODO: Check if circuit is empty
+        for op in circuit.all_operations():
+            qubits = [obj.x for obj in op.qubits]  # type: ignore
+            op_str = get_op_string(op.gate)  # type: ignore
+            seq_list.append([op_str, op.gate.exponent, qubits])  # type: ignore
+        json_list = json.dumps(seq_list)
+        return json_list
+
+    def _send_json(
+            self,
+            json_str: str,
+            id_str: Union[str, uuid.UUID],
+            remote_host: str = 'http://localhost:5000',
+            access_token: str = '',
+            repetitions: int = 1,
+            no_qubit: int = 1,
+    ):
+        """Sends the json string to the remote AQT device
+        Args:
+            json_str: json representation of the circuit
+            id_str: Unique id of the datapoint
+            remote_host: address of the remote device
+            repetitions: Number of repetitions
+            no_qubit: Number of qubits present in the device
+
+        Returns:
+            measurement results as an array of boolean
+        """
+        data = put(remote_host,
+                   data={
+                       'data': json_str,
+                       'id': id_str,
+                       'repetitions': repetitions,
+                       'no_qubits': no_qubit
+                   }).json()
+        while data['status'] != 'finished':
+            time.sleep(1.0)
+            data = put(remote_host,
+                       data={
+                           'data': json_str,
+                           'id': id_str,
+                           'acccess_token': access_token,
+                           'repetitions': repetitions,
+                           'no_qubits': no_qubit
+                       }).json()
+        measurements_int = data['samples']
+        measurements = np.zeros((len(measurements_int), no_qubit))
+        for i, result_int in enumerate(measurements_int):
+            for j in range(no_qubit):
+                measurements[i, j] = np.floor(result_int / 2**j)
+        # TODO: Check Big endian/little endian encoding!
```

Comment:
Seems important to do now

Reply:
This was to keep consistency with our backend that is not included in cirq. This has been resolved.